projects/addon-commerce/utils/format-currency.ts

```diff
@@ -4,9 +4,9 @@ import {padStart} from '@taiga-ui/cdk';
 import {getCurrencySymbol} from './get-currency-symbol';
 
 export function formatCurrency(currency: TuiCurrencyVariants): string {
-    const stringifiedCurrency = stringifyCurrency(currency);
+    const stringifiesCurrency = stringifyCurrency(currency);
```

Comment:
stringified is the correct spelling, it means turned into a string.

Reply:
fixed